core/app/views/refinery/admin/_menu.html.erb

```diff
@@ -1,4 +1,25 @@
-<nav id="menu">
-  <%= render partial: "refinery/admin/menu_item",
-             collection: current_refinery_user.active_plugins.in_menu -%>
-</nav>
+<button type="button" class="btn btn-primary sidebar-left-opener fixed-t-l z-5 hidden-sm-up"><i class="icon icon-bars"></i></button>
+
+<section id="sidebar-left" class="sidebar left inverted cms-ui">
+  <div class="sidebar-top-nav">
+    <button type="button" class="btn btn-white sidebar-left-opener abs-t-l z-5 hidden-sm-up"><i class="icon icon-close"></i></button>
+    <div class="site-name">
+      <% if content_for?(:custom_sidebar_header_link) %>
+        <%= yield :custom_sidebar_header_link %>
+      <% else %>
+        <i class="icon icon-bars"></i> <%= link_to Refinery::Core.site_name, '/', :class => 'ellipsis' %>
+      <% end %>
+    </div>
+  </div>
+  <div id="sidebar-left-inner" class="sidebar-inner">
+    <ul class="list-unstyled">
+      <%= render partial: "refinery/admin/menu_item",
+                 collection: current_refinery_user.active_plugins.in_menu -%>
+    </ul>
+  </div>
+  <%= link_to (t('.log_out', site_bar_translate_locale_args) + ' <i class="icon icon-log-out"></i>').html_safe,
```

Comment:
Sorry, I think any commit was probably not intended to escape my local
environment.

-- 
Anita Graham
anita@joli.com.au

From:  Stefan Spicer <notifications@github.com>
Reply-To:  refinery/refinerycms
<reply+00060f3e755d0eb88e2c78482fce9089caf26b825e7fba3992cf000000011339f1979
2a170ce03a9ab76@reply.github.com>
Date:  Thursday, 28 April 2016 at 11:43 PM
To:  refinery/refinerycms <refinerycms@noreply.github.com>
Cc:  Anita Graham <anita@joli.com.au>, Mention <mention@noreply.github.com>
Subject:  Re: [refinery/refinerycms] Glass menu sidebar (#3167)


In core/app/views/refinery/admin/_menu.html.erb
<https://github.com/refinery/refinerycms/pull/3167#discussion_r61451126> :
> +    <button type="button" class="btn btn-white sidebar-left-opener abs-t-l
z-5 hidden-sm-up"><i class="icon icon-close"></i></button>
> +    <div class="site-name">
> +      <% if content_for?(:custom_sidebar_header_link) %>
> +        <%= yield :custom_sidebar_header_link %>
> +      <% else %>
> +        <i class="icon icon-bars"></i> <%= link_to Refinery::Core.site_name,
'/', :class => 'ellipsis' %>
> +      <% end %>
> +    </div>
> +  </div>
> +  <div id="sidebar-left-inner" class="sidebar-inner">
> +    <ul class="list-unstyled">
> +      <%= render partial: "refinery/admin/menu_item",
> +                 collection: current_refinery_user.active_plugins.in_menu -%>
> +    </ul>
> +  </div>
> +  <%= link_to (t('.log_out', site_bar_translate_locale_args) + ' <i
class="icon icon-log-out"></i>').html_safe,
@anitagraham <https://github.com/anitagraham>  I'm not following what this
action_icon helper helps with, so I thought I'd give it a try to understand.
I got this error which confuses me even more.
Started GET "/admin/login" for 127.0.0.1 at 2016-04-28 08:30:16 -0700
[...]
Completed 500 Internal Server Error in 5457ms (ActiveRecord: 4.3ms)

ActionView::Template::Error (".logout_icon" failed to @extend ".fa-logout".
The selector ".fa-logout" was not found.
Use "@extend .fa-logout !optional" if the extend should be able to fail.
):
    4:   <%= yield :meta %>
    5:   <title><%= Refinery::Core.site_name -%> - Refinery</title>
    6:   <%= csrf_meta_tags %>
    7:   <%= stylesheet_link_tag 'refinery/refinery' %>
    8:   <% visual_editor_stylesheets.each do |css| %>
    9:     <%= stylesheet_link_tag css.path, css.options %>
   10:   <% end %>
  
/home/stefan/dev/rails/refinerycms/core/app/assets/stylesheets/refinery/mixi
ns/_icon.scss:4

Using a different icon instead (temporarily) produced this HTML which
doesn't have the "Log out" text on the link:
<a class="user_comment_icon" href="/admin/logout" tooltip="Log out"></a>
So then I tried action_label instead and the text shows up, but not the
icon, and there is a tooltip in the way (it shouldn't be there).

Sorry, either I'm missing something or this isn't going to work for this
instance.


You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/refinery/refinerycms/pull/3167/files/12843b9186462b95390
6bffcd1617ca7c65ee1df#r61451126>